db/blip_handler.go

```diff
@@ -968,6 +979,11 @@ func (bh *blipHandler) sendGetAttachment(sender *blip.Sender, docID string, name
 	if isCompressible(name, meta) {
 		outrq.Properties[BlipCompress] = "true"
 	}
+
+	if bh.blipContext.ActiveProtocol() == BlipCBMobileReplicationV3 {
```

Comment:
ActiveProtocol is a relatively expensive call (it's doing strings.SplitN every time), so I don't think it makes sense to call it more than once per blipContext.  Can we store this on the context closer to context initialization?